src/olympia/versions/utils.py

```diff
@@ -52,12 +52,15 @@ def write_svg_to_png(svg_content, out):
         return convert_svg_to_png(temporary_svg.name, out)
 
 
+SVG_DIMENSIONS_REGEX = rb'(?=.* width="(?P<width>\d+)")(?=.* height="(?P<height>\d+)")'
+
+
 def encode_header(header_blob, file_ext):
     try:
         if file_ext == '.svg':
-            tree = lxml.etree.fromstring(header_blob)
-            width = int(tree.get('width'))
-            height = int(tree.get('height'))
+            dimensions = re.search(SVG_DIMENSIONS_REGEX, header_blob).groupdict()
```

Comment:
do we have test coverage for this?

Reply:
well, some tests failed when I made a mistake 😁

Reply:
Ok. I was wondering what would happen if a svg has more than one tag with `width`/`height` attributes but `re.search()` only returns the first occurrence, so that's good: https://docs.python.org/3/library/re.html#re.search